README.md

```diff
@@ -20,40 +41,9 @@ sbt:transporter> test
 It's also possible to run individual `sbt` commands directly from bash. This is _not_ the recommended way to use `sbt`,
 as you'll eat the tool's (nontrivial) startup costs on every command.
 
-## Running
-For now, Transporter only runs locally.
-
-### Manager service
-Launch Transporter's manager service using:
-```bash
-$ sbt
-sbt:transporter> transporter-manager/run
-```
-
-Using default configuration the service will bind to port `8080`, and serve interactive API documentation at the
-[root path](http://localhost:8080/). To successfully execute requests, the manager needs:
-
-  1. PostgreSQL running locally on port 5432, with a default `postgres` database.
-  2. Kafka running locally, with a bootstrap server bound to port `9092`.
-
-On OS X, these services can be installed using `brew`:
-```bash
-# Will also install zookeeper, if not present:
-$ brew install postgresql@9.6 kafka
-# Replace 'run' with 'start' below to make the services auto-start on login:
-$ brew services run postgresql@9.6 &&
-  brew services run zookeeper &&
-  brew services run kafka
-```
-
-Finally, connecting to the DB requires filling out config in the manager's `application.conf`. For OS X's `brew`
-installation with default settings, the config should be:
-```bash
-$ cat <<-EOF > manager/src/main/resources/application.conf
-org.broadinstitute.transporter.db.username = "$(whoami)"
-org.broadinstitute.transporter.db.password = ""
-EOF
-```
+## Running Locally
+1. Run the [Manager](manager/README.md)
+2. Run an [agent](agents/README.md)
```

Comment:
same for agents 

https://github.com/DataBiosphere/transporter/blob/dm-transporter-docs-DSPDC-284/agents/README.md#running-locally

Reply:
Note that the links are for m-transporter-docs-DSPDC-284 branch